storyscript/Hub.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from functools import lru_cache
+
+from storyhub.sdk.StoryscriptHub import StoryscriptHub
+
+
+@lru_cache(maxsize=1)
+def _story_hub():
+    """
+    Cached instance of the hub sdk
+    """
+    return StoryscriptHub()
+
+
+def get_story_hub_instance():
```

Comment:
```suggestion
def story_hub_instance():
```

or maybe just `story_hub`?
Also, the private method has a docstring, but this one doesn't.